docs/community_resources/phpstorm_plugin.md

```diff
@@ -0,0 +1,173 @@
+
+You can find details of the installation procedure in Composer log window.
+
+### File templates
+
+The plugin provides the following built-in file templates:
+
+| Name | Comment |
+|---|---|
+| Back Office tab | Class implementing `EzSystems\EzPlatformAdminUi\Tab\AbstractTab` |
+| Block event subscriber | Event subscriber for `BlockRenderEvents::getBlockPreRenderEventName(...)` event |
+| Composite Criterion | Criterion class based on `\eZ\Publish\API\Repository\Values\Content\Query\Criterion\CompositeCriterion` |
+| Field definition form mapper | Class implementing `EzSystems\EzPlatformAdminUi\FieldType\FieldDefinitionFormMapperInterface` |
+| Field Type | Field Type class based on `eZ\Publish\SPI\FieldType\Generic\Type` |
+| Field Type Comparable | Class implementing `EzSystems\EzPlatformVersionComparison\FieldType\Comparable` |
+| Field Type Indexable | Class implementing `eZ\Publish\SPI\FieldType\Indexable` |
+| Field value form mapper | Class implementing `EzSystems\EzPlatformContentForms\FieldType\FieldValueFormMapperInterface` |
+| Field value object | Field Type value class |
+| Menu configuration event subscriber | Event subscriber for `EzSystems\EzPlatformAdminUi\Menu\Event\ConfigureMenuEvent::MAIN_MENU`  |
+| Policy provider | Class implementing `eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Security\PolicyProvider\PolicyProviderInterface` |
+| Policy provider (YAML) | Policy provider class based on `eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Security\PolicyProvider\YamlPolicyProvider` |
+| Query Type | Query Type class based on `eZ\Publish\Core\QueryType\OptionsResolverBasedQueryType` |
+| Schema builder subscriber | Event subscriber for `EzSystems\DoctrineSchema\API\Event\SchemaBuilderEvent::BUILD_SCHEMA` event |
+| SiteAccess-aware configuration | SiteAccess-aware configuration definition class based on `eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser` |
+| Value object input parser | REST input parser class based on `EzSystems\EzPlatformRest\Input\BaseParser` |
+| Value object visitor | REST value visitor class based on `EzSystems\EzPlatformRest\Output\ValueObjectVisitor` |
+| Workflow action listener | Workflow action listener class based on `EzSystems\EzPlatformWorkflow\Event\Action\AbstractTransitionWorkflowActionListener` |
+
+The templates are available in, for example, the context menu in **Project window** > **New** > **Ibexa DXP**.
+
+The list of available file templates depends on the Ibexa DXP edition used by the project.
+
+For all file templates you can customize:
+
+- class name
+- class namespace
+- file name
+- directory
+
+![File template](img/phpstorm_plugin_file_template.png)
+
+To customize file templates, go to **File** > **Settings** > **Editor** > **File and Code templates**.
```

Comment:
Again, navigation differs between platforms.
We could say "On Ubuntu ..., on Windows..., and on Mac ..."